### PR TITLE
Create render method for Board class

### PR DESCRIPTION
**Pull request by mkrumholz**
## What does this change?

render method

## What does this fix?

Renders board with cells rendered (empty tests completed)

## Is this a bug or a feature? 

feature

## How was this tested?

Tests written for empty board to render properly

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -117,4 +117,19 @@ def place(ship, placement_coordinates)
     end
     #may need to add error/return value for non placement
   end
+
+  def render
+    rendered_cells = @cells.map do |coordinate, cell|
```
Comment (suzkiee) [inline lib/board.rb:122]: Could each of these enumerable methods be a helped method outside?

File: lib/board.rb [hunk 2]
```diff
@@ -117,4 +117,19 @@ def place(ship, placement_coordinates)
...
+      cell.render
+    end
+    row_arrays = rendered_cells.each_slice(4).to_a
+    rows = row_arrays.map do |array|
+      array.join(' ')
+    end
+
+    range = "A".."D"
+    row_letters = range.to_a
```
Comment (suzkiee) [inline lib/board.rb:131]: Maybe the heaviest lift that the render method should be the printing of board/ship placements?

File: lib/board.rb (hunk 1 above)
Comment (mkrumholz) [reply lib/board.rb:122]: Yes! I think we definitely need to do that in the next round.

File: lib/board.rb (hunk 2 above)
Comment (mkrumholz) [reply lib/board.rb:131]: Yep, definitely agreed

**Pull request merged**